samples/interop/Direct3DInteropSample/Direct3DInteropSample.csproj

```diff
@@ -1,31 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>net461</TargetFramework>
-    </PropertyGroup>
-    <ItemGroup>
-        <PackageReference Include="SharpDX.Mathematics" Version="4.0.1" />
-        <PackageReference Include="SharpDX.D3DCompiler" Version="4.0.1" />
-        <Compile Update="**\*.paml.cs">
-            <DependentUpon>%(Filename)</DependentUpon>
-        </Compile>
-        <EmbeddedResource Include="**\*.paml">
-            <SubType>Designer</SubType>
-        </EmbeddedResource>
-    </ItemGroup>
-    <ItemGroup>
-      <None Remove="MiniCube.fx" />
-    </ItemGroup>
-    <ItemGroup>
-      <EmbeddedResource Include="MiniCube.fx">
-        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      </EmbeddedResource>
-    </ItemGroup>
-    <ItemGroup>
-        <ProjectReference Include="..\..\..\src\Windows\Avalonia.Direct2D1\Avalonia.Direct2D1.csproj" />
-        <ProjectReference Include="..\..\..\src\Windows\Avalonia.Win32\Avalonia.Win32.csproj" />
-        <ProjectReference Include="..\..\MiniMvvm\MiniMvvm.csproj" />
-    </ItemGroup>
-    <Import Project="..\..\..\build\Rx.props" />
-    <Import Project="..\..\..\build\ReferenceCoreLibraries.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
why `2.0`? Other examples use `netcoreapp3.1`

Reply:
This is copied from NativeEmbedSample, which uses `<TargetFramework>netcoreapp2.0</TargetFramework>`

Reply:
probably it makes sense to bump it there as well

Reply:
I bumped both to `netcoreapp3.1`.